bin/install

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+cd "$(dirname "$0")/.."
+
+mkdir -p app
+git clone git@github.com:uc-group/daffy.git app
```

Comment:
Git clone should create the app folder if it does not exist. How about adding app folder with .gitignore to the project structure so there wouldn't be any problems when someone starts first the containers that create volume and than the install script.

Reply:
Sure, we can do this that way

Reply:
My bad, when cloning the app folder shouldn't be empty.